lib/generators/manageiq/plugin/templates/spec/spec_helper.rb

```diff
@@ -5,3 +5,5 @@
 
 Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
 Dir[File.join(__dir__, "support/**/*.rb")].each { |f| require f }
+
+require "<%= plugin_name %>"
```

Comment:
@Fryguy how does this work with the `.tr` of / to space?  Am I misreading?

Reply:
plugin_name gives the dasherized version... e.g. `require "manageiq-providers-whatever"`.  The new method introduced is called `plugin_human_name`, but that's used elsewhere.